features/partners.feature

```diff
@@ -4,128 +4,231 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
   Van een inschreven persoon worden -indien gevraagd met de fields parameter- het (de) huwelijk(en) of geregistreerd partnerschap(pen) geleverd.
   Dit bevat de gegevens over de relatie (huwelijk of partnerschap) plus enkele identificerende eigenschappen van de persoon waarmee het huwelijk of partnerschap is aangegaan.
 
-  Rule: Alleen een actueel huwelijk of geregistreerd partnerschap wordt teruggegeven.
-    Een huwelijk of partnerschap is actueel wanneer het niet is ontbonden: groep 07 Ontbinding huwelijk/geregistreerd partnerschap komt niet voor (heeft geen waarde)
+  # RvIG levert personen in de vorm GbaPersoon, waarin alleen onbewerkte GBA-V gegevens worden geleverd
+  # Dit betekent dat in 'partners' zowel actuele partners zitten als ontbonden partners
+  # Een proxy vertaalt dit naar de vorm Persoon, waarin sommige gegevens in bewerkte vorm worden opgenomen en waarin informatievragen kunnen zitten
+  # Een voorbeeld van een informatievraag is naam.aanschrijfwijze, waarin op basis van de aanduiding naamgebruik, de naam van de persoon en de naam van de (ex)partner een aanschrijfnaam wordt samengesteld
+  # De proxy verwijdert de partners met datum ontbinding die geleverd zijn in de GbaPersoon
 
-    Scenario: Actueel huwelijk
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 999990184 |
-      En de persoon heeft een partner met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 999992935 |
-      En de partner heeft de volgende naam gegevens
-      | naam              | waarde |
-      | voornamen (02.10) | Merel  |
-      En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
-      | naam                      | waarde   |
-      | datum aangaan (06.10)     | 20010215 |
-      | soort verbintenis (15.10) | H        |
-      En de partner heeft GEEN ontbindingHuwelijkPartnerschap gegevens
+  @gba
+  Rule: Het gegeven 'partners' in GbaPersoon bevat ook ontbonden huwelijken en partnerschappen
+
+    @gba
+    Scenario: een actuele partner en geen ontbonden partner(s)
+      Gegeven de persoon met burgerservicenummer 999992934 heeft de volgende huwelijken/partnerschappen in de registratie
```

Comment:
Bij gegeven staan nu de waardetabellen niet gedefinieerd.

Reply:
ik verwijder de tabelwaarden uit het scenario. het doel is hier niet te tonen hoe een code vertaald wordt naar omschrijving, het doel is te tonen dat zowel actuele als ontbonden relaties worden opgenomen